hack/deployer/runner/ocp.go

```diff
@@ -186,7 +167,54 @@ func (d *OcpDriver) Execute() error {
 	return err
 }
 
-func (d *OcpDriver) clusterExists() (bool, error) {
+func (d *OcpDriver) ensureContext() (cleanUp func(), err error) {
+	cleanUp = func() {} // NOOP
+	if d.ctx["WorkDir"].(string) == "" {
```

Comment:
Is this guaranteed to be not nil? Otherwise the cast would panic.

Reply:
No it's not.